conf_files/pocs.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: Generic PANOPTES Unit
-# pan_id: PANxyz   # This unit's assigned identifier.
+pan_id: PAN000   # This unit's assigned identifier.
```

Comment:
... PAN000 is a "fake" id that no unit uses.

Reply:
I greatly expanded this.